modules/@apostrophecms/asset/index.js

```diff
@@ -6,18 +6,6 @@ const globalIcons = require('./lib/globalIcons');
 const path = require('path');
 const express = require('express');
 
-// Wrap webpack to report its errors readably, in
-// preparation to promisify it
-
-function webpack(config, cb) {
```

Comment:
No longer necessary because webpackModule's function signature is now correct to pass directly to promisify.